drivers/wifi/nrf700x/zephyr/src/zephyr_wpa_supp_if.c

```diff
@@ -8,8 +8,6 @@
  * @brief File containing WPA supplicant interface specific definitions for the
  * Zephyr OS layer of the Wi-Fi driver.
  */
-
-#ifdef CONFIG_WPA_SUPP
 #include <stdlib.h>
```

Comment:
Not related to these changes in this file, but related to this file in general and could be handled as a separate requirement if seen feasible.
Instead of device_get_binding("wlan0"), I would like to get a wifi device as following:
https://github.com/nrfconnect/sdk-nrf/blob/main/lib/location/method_wifi.c#L278

I.e. use a device tree to specify chosen Wi-Fi device like done in: https://github.com/nrfconnect/sdk-nrf/blob/main/samples/nrf9160/location/esp_8266_nrf9160ns.overlay#L58

Reply:
Sure, will check this, thanks.